runtimes/kotlin-1.6/src/main/kotlin/io/openruntimes/kotlin/RuntimeContext.kt

```diff
@@ -0,0 +1,42 @@
+package io.openruntimes.kotlin
+
+import kotlin.Any
+import kotlin.collections.ArrayList
+import kotlin.collections.arrayListOf
+
+import com.google.gson.GsonBuilder
+import com.google.gson.Gson
+
+class RuntimeContext(req: RuntimeRequest, res: RuntimeResponse) {
+    var req: RuntimeRequest
+    var res: RuntimeResponse
+
+    var logs: ArrayList<String>
+    var errors: ArrayList<String>
+
+    init {
+        this.req = req;
+        this.res = res;
+        this.logs = arrayListOf<String>()
+        this.errors = arrayListOf<String>()
+    }
+
+    fun log(message: Any) {
+        if(message is Map<*, *> || message is List<*>) {
+            var gson: Gson = GsonBuilder().serializeNulls().create()
```

Comment:
`gson` can be part of a companion:

```kotlin
class RuntimeContext {
  companion object {
      private val gson = GsonBuilder.serializeNulls().create()
  }
}
```

Reply:
I cannot do that. Gives me errors `Unresolved reference: serializeNulls`. Any tips?

Reply:
Ah yep syntax error in my snippet, needs to be `GsonBuilder()` instead of `GsonBuilder` :+1: